thanks.md

```diff
@@ -2,5 +2,17 @@
 
 In this file, you can read about where and how people use Rubberduck:
 
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
-- Edit this file, and tell us why you like Rubberduck. - [@username](https://github.com/username)
```

Comment:
I think we could use a `<!-- add new content above -->` comment here (we want the most-recent on top, right?)

Reply:
seems like a good suggestion. We might even add a template there :)